catkin_home/src/action_selectors/scripts/DeepSpeech/deploy/ros_server.py

```diff
@@ -6,6 +6,8 @@
 import argparse
 import functools
 import paddle.v2 as paddle
+# TODO: Check if rospy should also be imported.
```

Comment:
Well yes, it does. I only was wondering if it needs it because this file imports `rospkg` and I had the doubt if it needs to have imported/initiated `rospy`. Because anyway, this file is not the main node, it is called by an actual node that imports `rospy`.